sys-cluster/gearmand/gearmand-0.34-r1.ebuild

```diff
@@ -51,6 +51,9 @@ src_configure() {
 		# preprocessor then.
 		append-cppflags -DDEBUG
 	fi
+	
```

Comment:
If you did `git log -p`, you'd notice red block saying you've introduced whitespace issue here. But I'm going to fix it for you.